File: 33_SearchElement.js


var search = function(nums, target) {
    
    return nums.indexOf(target);
};


let main = function() {

    let nums = [4,5,6,7,0,1,2], target = 9;
    let result = search(nums,target);
    
    console.log(result);
}

main();

File: 33_SearchElement.c
#include<stdio.h>

int search(int* nums, int numsSize, int target) 
{
    int iPos = -1;
    for(int i = 0; i < numsSize; i++)    
    {
        if(nums[i] == target)
        {
            iPos = i;
            break;
        }
    }
    return iPos;
}

int main()
{


    return 0;
}